perception_dataset/deepen/non_annotated_t4_to_deepen_converter.py

```diff
@@ -145,48 +146,50 @@ def _convert_one_frame(self, input_dir: str, output_dir: str, frame_index: int):
         config_data.save_json(output_dir)
 
     @staticmethod
-    def _get_camera_token(camera_channel: str, sample, nusc) -> str | None:
+    def _get_camera_token(camera_channel: str, sample, t4_dataset) -> str | None:
         """Get camera token for `camera_channel` in the given `sample` data from a NuScenes dataset.
         Args:
             camera_channel (str): Camera channel name e.g. CAM_FRONT to look for
-            sample: Sample data for a specific frame from NuScenes = `nusc.sample[frame_index]`
-            nusc (NuScenes): NuScenes dataset
+            sample: Sample data for a specific frame from NuScenes = `t4_dataset.sample[frame_index]`
+            t4_dataset (Tier4): Tier4 dataset
         Return:
             camera_token: str | None
                 None if not found
         """
         camera_token: str | None = None
-        if camera_channel in sample["data"].keys():
-            camera_token = sample["data"][camera_channel]
+        if camera_channel in sample.data.keys():
+            camera_token = sample.data[camera_channel]
         else:
-            sample_data = [s for s in nusc.sample_data if s["sample_token"] == sample["token"]]
+            sample_data = [s for s in t4_dataset.sample_data if s.sample_token == sample.token]
             for sensor in sample_data:
-                if sensor["channel"] == camera_channel:
-                    camera_token = sensor["token"]
+                if sensor.channel == camera_channel:
+                    camera_token = sensor.token
                     break
         return camera_token
 
-    def _get_data(self, nusc: NuScenes, sensor_channel_token: str) -> Dict[str, Any]:
-        sd_record = nusc.get("sample_data", sensor_channel_token)
-        cs_record = nusc.get("calibrated_sensor", sd_record["calibrated_sensor_token"])
-        ep_record = nusc.get("ego_pose", sd_record["ego_pose_token"])
+    def _get_data(self, t4_dataset: Tier4, sensor_channel_token: str) -> Dict[str, Any]:
+        sd_record: SampleData = t4_dataset.get("sample_data", sensor_channel_token)
+        cs_record: CalibratedSensor = t4_dataset.get(
+            "calibrated_sensor", sd_record.calibrated_sensor_token
+        )
+        ep_record: EgoPose = t4_dataset.get("ego_pose", sd_record.ego_pose_token)
 
         sensor2ego_transform = transform_matrix(
-            translation=cs_record["translation"],
-            rotation=Quaternion(cs_record["rotation"]),
+            translation=cs_record.translation,
+            rotation=Quaternion(cs_record.rotation),
         )
         ego2global_transform = transform_matrix(
-            translation=ep_record["translation"],
-            rotation=Quaternion(ep_record["rotation"]),
+            translation=ep_record.translation,
+            rotation=Quaternion(ep_record.rotation),
```

Comment:
[NITS] For the same reason above
```suggestion
            rotation=ep_record.rotation,
```